<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvironmentFileAssociationWidget</class>
 <widget class="QWidget" name="EnvironmentFileAssociationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Open Each File In</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbOpenInMultiApplication">
        <property name="text">
         <string>Independent Red Panda C++ applications</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbOpenInSingleApplication">
        <property name="text">
         <string>The same Red Panda C++ application</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File Types:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListView" name="lstFileTypes">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Just check or uncheck for which file types Red Panda C++ wil be registered as the default application to open them ... </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
